keep-ui/entities/workflows/model/types.ts

```
@@ -19,23 +20,264 @@ export type DefinitionV2 = {
+  V2StepConditionAssertSchema.partial({ id: true }),
+  V2StepConditionThresholdSchema.partial({ id: true }),
+  V2StepForeachSchema.partial({ id: true }),
+]);
+
+export type V2StepTemplate = z.infer<typeof V2StepTemplateSchema>;
+
+export type V2StartStep = {
+  id: "start";
+  type: "start";
+  componentType: "start";
+  properties: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)